Dockerfiles/agent/entrypoint/60-network-check.sh

```diff
@@ -8,5 +8,5 @@ if [[ -n "${ECS_FARGATE}" ]]; then
 fi
 
 if [[ ! -d /host/proc ]]; then
-    rm /etc/datadog-agent/conf.d/network.d/conf.yaml.default
+    rm /etc/datadog-agent/conf.d/network.d/conf.yaml.default || true
```

Comment:
Or if you want to silent it with `rm -f /etc/datadog-agent/conf.d/network.d/conf.yaml.default`?

Reply:
@christofferh I'm gonna merge as is and do a follow-up PR to switch to `-f` while fixing a few other things in the entrypoint 👍